src/commonmark/en/content/developer/web-api/new-tracker.md

```diff
@@ -890,7 +890,7 @@ There are various error codes for different error scenarios. The following table
 | E1070 | Could not find OrganisationUnit: `{0}`, linked to Enrollment. | The system could not find the OrganisationUnit specified in the Enrollment payload. |
 | E1074 | FeatureType is missing. | |
 | E1075 | Attribute: `{0}`, is missing uid. | |
-| E1076 | Attribute: `{0}`, value is null. | |
+| E1076 | `{0}` `{1}` is mandatory and can''t be null | |
```

Comment:
As mentioned in the other PR; We should update the typo in this PR first, and then in the code.

Reply:
I don't really like the use of abbreviations like "can't" in formal documents. We should use "cannot" instead.
However, it is used in several places in this document already, so it can be corrected independently of this PR.